astro/src/content/docs/apis/_login-id-field.mdx

```diff
@@ -0,0 +1,22 @@
+import APIField from 'src/components/api/APIField.astro';
+import InlineField from 'src/components/InlineField.astro';
+import Aside from 'src/components/Aside.astro';
+
+<APIField name="loginId" type="String" required={!props.optional}>
```

Comment:
this and new type component give us a better chance at getting it accurate, in 1 place